.travis.yml

```diff
@@ -0,0 +1,14 @@
+sudo: required
+cache:
+  directories:
+    - $HOME/.m2
+language: java
+jdk:
+  - oraclejdk8
+install: true
+script:
+  - '[ "${TRAVIS_PULL_REQUEST}" != "false" ] || ./mvnw package -Pfull -s .settings.xml -U -Dmaven.test.redirectTestOutputToFile=true'
+  - '[ "${TRAVIS_PULL_REQUEST}"  = "false" ] || ./mvnw package -Pfull -s .settings.xml -U -Dmaven.test.redirectTestOutputToFile=true'
```

Comment:
Why do we need these two identical commands?
Looks like both one of them is performed independently of the `TRAVIS_PULL_REQUEST` value.

Or am I missing anything?